src/cli.js

```diff
@@ -35,7 +52,6 @@ async function getSchemaUrlForFilename(filename, cache, ttl) {
       }
     }
   });
-
   if (matches.length === 1) {
     return matches[0].url;
```

Comment:
I have been pretty slack so far about assuming the structure of the schemastore catalog, but once the catalog can be arbitrary user-supplied JSON I think we need to be more defensive about dealing with the possibility that the schema catalog might not be valid. One option is that (of course :) ) there is a JSON schema for validating schema catalogs https://json.schemastore.org/schema-catalog.json so we could check the supplied catalog against that before we try to use it, although the downside of that is it requires a bunch of fields we're not actually going to use. We should a least consider the possibility that the catalog might not match our assumptions (contains a key called `schemas` which is an array[] of objects{} which must have a `.url` property. There is also another reason why maybe we don't want to strictly validate that a user-supplied catalog matches https://json.schemastore.org/schema-catalog.json which we will come onto in the next comment..

Reply:
Fixed by dad46a136bc280a1863005a941aa9363e03e0a28